apps/backend/Dockerfile

```diff
@@ -7,8 +7,10 @@ RUN mkdir -p /home/node/app
 WORKDIR /home/node/app
 
 COPY --chown=node:node package*.json ./
+COPY openapi.yaml openapi.yaml
```

Comment:
If possible can we move this file to docker-orchestration or busapp settings .If possible we can load it up as config maps.

Reply:
I'm not sure we can but happy to be proved worng. Right now the main 2 issues I see are:
- The linter doesn't pass without the generated files meaning the linter fails for anyone who would not have access to docker-orchestration or busapp settings like github actions or other facilitys working on this project
- The test would also fail on github actions for the same reason as above

Reply:
That must be true the only other way is to change this into a package but all depends what ELI and ESS say l guess.

Reply:
If the problem is that the linter doesn't like the generated files, we just need to tell it to ignore them by adding the generated directory to the `.eslintignore` file.

Reply:
It is a problem that were functions from the generated files are used. Like the `STFCUserDataSource` doesnt type check if the gerenerated files dont exist.